package.json

```diff
@@ -33,7 +33,7 @@
     "@vue/cli-plugin-babel": "^3.4.1",
     "@vue/cli-plugin-typescript": "^3.4.1",
     "@vue/cli-service": "^3.4.1",
-    "node-sass": "^4.11.0",
+    "node-sass": "^4.12.0",
```

Comment:
^ This symbol will download the latest version 4.13.0 in 4.x.

Reply:
but when use ^4.11.0 , it only download 4.11.0 , the download the latest version. so it download the node-sass from https://github.com/sass/node-sass/releases/download/v4.11.0/linux-x64-72_binding.node,  the url  is not exit now.

Reply:
Check the version lock file, I think that is the reason. https://github.com/apache/skywalking-rocketbot-ui/blob/master/package-lock.json#L7257

Reply:
I just confirmed the meaning of these symbols. ^ will  update the current library version to the latest version of the current major version (that is, the first digit). ~ will update to the latest version of the current minor version (that is, the number in the middle). in package.json  , we use ^4.12.0, it will find the latest version , that is 4.13.0 .  and i found the tag  version track of the node-sass, find  it has v4.11.0,v,v4.12.0,v4.13. I guess if the tag v affects the automatic update of the node-sass version? I can't confirm this. But overall, I think it would be okay to use v4.13.0.

Reply:
https://github.com/sass/node-sass/tags

Reply:
pls recheck your version lock file, package-lock.json or yarn.lock. Maybe your version is 4.11.0 in lock file.
<img width="249" alt="屏幕快照 2019-11-13 上午11 22 13" src="https://user-images.githubusercontent.com/20871783/68730538-ef723b00-0607-11ea-9d31-5be9f16f3dcd.png">
It's mine.

Reply:
i check it, when i run npm install, it will modify the version from 4.11.0 to 4.13.0 auto in package-lock.json. shou i modify the package-lock.json file to 4.13.0  and push it?

Reply:
I think you should update the version in package-lock.json to fix the bug.

Reply:
i push the package-lock.json.

Reply:
As you have updated the lock JSON, this doesn't need to change, right?

Reply:
yes, i think this is not need to change , but @Fine0830 think i need to update this file.

Reply:
I think lock file is our modules version we use now. It needs to update. package.json is the version range of the module we are using,  it don't need to change.

Reply:
@a198720 Could you fix this?